HandleNHDeleteInstallationCallOrchestrator/handler.ts

```diff
@@ -24,51 +40,85 @@ export type NhDeleteInstallationOrchestratorCallInput = t.TypeOf<
   typeof NhDeleteInstallationOrchestratorCallInput
 >;
 
-const logError = (
+const logPrefix = `NhDeleteInstallationOrchestratorCallInput`;
+
+function* deleteInstallation(
   context: IOrchestrationFunctionContext,
-  logPrefix: string,
-  errorOrNHCallOrchestratorInput
-) => {
-  context.log.error(`${logPrefix}|Error decoding input`);
-  context.log.verbose(
-    `${logPrefix}|Error decoding input|ERROR=${readableReport(
-      errorOrNHCallOrchestratorInput.value
-    )}`
+  input: NHDeleteInstallationActivityInput,
+  retryOptions: df.RetryOptions
+): Generator<Task, "SUCCESS"> {
+  const activityName = "HandleNHDeleteInstallationCallActivity";
```

Comment:
woulnd't it be better to import this from the activity module? (otherwise if you refactor the name there you break the orchestrator here).

Reply:
I tried, but it seems there is some code loading issue that cannot allow us to be sure that the value imported has already been loaded.. I saw many crashes in my local environment

Reply:
what do you mean with "already been loaded" ? if you import it it's just a module reference

Reply:
That's exactly what I thought... I saw those strange errors.. and since everybody was passing it as a plain string I just thought that there were no other way to do it.. but let me try one more time

Reply:
It's working fine now, actually I really don't remember the issue I've experienced, but maybe I just confused some retry error with issues related to that change -.-"